packages/template-retail-react-app/app/pages/checkout/confirmation.test.js

```diff
@@ -26,6 +26,13 @@ jest.mock('../../commerce-api/auth', () => {
     }
 })
 
+jest.mock('../../utils/url', () => {
+    return {
+        ...jest.requireActual('../../utils/url'),
+        removeSiteLocaleFromPath: jest.fn()
+    }
+})
+
```

Comment:
```suggestion
```
This makes the test pass, but it does so by not actually testing the code, so it doesn't really solve the problem. 😕

Reply:
yes, i know. see same conversation here. 

https://github.com/SalesforceCommerceCloud/pwa-kit/commit/f8f4504f9938967b9e4252264dcf5e51ce401a33#r105815160

the test is failing on the navigate back to the account page where removeSiteLocaleFromPath is called.

The chain ultimately goes:
removeSiteLocaleFromPath -> getParamsFromPath -> absoluteUrl -> getAppOrigin

which ultimately breaks here with TypeError: Cannot read properties of null (reading '_location')

const getAppOrigin = () => {
  if (typeof window !== 'undefined') {
    return window.location.origin;
  }
  ...
}


no amount of mocking any of the above functions would work. I'm open to ideas. 🤷‍♀️

this is a minor bug that i'm trying to get in to support hybrid deployments and just trying to clear this failing test. I'll attempt more efforts later

Reply:
Given the nature of the bug fix and the fact that all of our tests are super flaky (not just this one), I think it's fine if you just disable the broken one with `test.skip` for now, and we'll re-enable it once we've fixed our testing setup.

Reply:
perfect, thanks @wjhsf, all set.